src/main/groovy/io/gatling/gradle/LogbackConfigTaskAction.groovy

```diff
@@ -13,6 +13,7 @@ class LogbackConfigTaskAction implements Action<Task> {
         </encoder>
         <immediateFlush>false</immediateFlush>
     </appender>
+    <logger name="io.gatling.http.engine.response" level="${logHttp.logLevel ?: logLevel}" />
```

Comment:
I think it would be better not to add a seperate logger at all, if logHttp is `NONE`.

```
if (logHttp != NONE) {
   add `io.gatling.http.engine.response` logger
}
```

Reply:
Will give it a try. Not familiar with Groovy yet.

Reply:
ok, let's proceed with current approach. 
we could fix it later.